### PR TITLE
1116489 vagrant

### PR DESCRIPTION
**Pull request by willkg**
Lots of fixes to our vagrant environment.

### CODE REVIEW

**Comment by willkg** [comment]
Travis doesn't test this at all. I'm going to land it because I claim it's better than what we currently have. Plus then @turtleloveshoes can test it out.

**Comment by turtleloveshoes** [comment]
I went through some of the tests in the documentation for this. 
1. $ mysql -u root -p
mysql> GRANT ALL ON test_kitsune.* TO kitsune@localhost IDENTIFIED BY '<YOUR_PASSWORD>';

Ran great
2. Not so much
vagrant@vagrant-ubuntu-trusty-64:~/kitsune$ ./manage.py test -s --noinput --logging-clear-handlers
Traceback (most recent call last):
  File "./manage.py", line 13, in <module>
    from kitsune.sumo.monkeypatch import patch  # noqa
  File "/home/vagrant/kitsune/kitsune/sumo/__init__.py", line 6, in <module>
    from kitsune.sumo.monkeypatch import patch  # noqa
  File "/home/vagrant/kitsune/kitsune/sumo/monkeypatch.py", line 1, in <module>
    from django.forms import fields
ImportError: No module named django.forms

Will follow up with other tests as well. 

To note add to documentation: 
Git pulls.

**Comment by willkg** [comment]
I moved the conversation to the comments in the bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1116489

File: bin/vagrant_provision.sh [hunk 1]
```diff
@@ -1,71 +1,113 @@
...
-# Copy configurations for kitsune and mysql
-# MySQL Default User: root
-# MySQL Default Pass: rootpass
-cp $INSTALL_DIR/kitsune/configs/vagrant/settings_local.py \
-   $INSTALL_DIR/kitsune/kitsune/settings_local.py
+# Install git
+apt-get install -y -q git
 
-cp $INSTALL_DIR/kitsune/configs/vagrant/my.cnf /etc/mysql/my.cnf
+# Install mysql
```
Comment (safwanrahman) [inline bin/vagrant_provision.sh]: I think kitsune uses ``mariadb server``. so we should install ``mariadb`` instead of ``mysql``

**Pull request merged**